opm/simulators/wells/BlackoilWellModelGeneric.cpp

```diff
@@ -605,23 +605,21 @@ checkGconsaleLimits(const Group& group,
 }
 
 template<class Scalar>
-bool BlackoilWellModelGeneric<Scalar>::
-checkGroupHigherConstraints(const Group& group,
-                            DeferredLogger& deferred_logger,
-                            const int reportStepIdx,
-                            const int max_number_of_group_switch)
+std::pair<int, int>
+BlackoilWellModelGeneric<Scalar>::
+getGroupFipnumAndPvtreg() const
 {
     // Set up coefficients for RESV <-> surface rate conversion.
     // Use the pvtRegionIdx from the top cell of the first well.
     // TODO fix this!
     // This is only used for converting RESV rates.
     // What is the proper approach?
+    // See discussion in: https://github.com/OPM/opm-simulators/issues/2921 for more details.
```

Comment:
In the event that this project is ever moved that link might not be valid or people might be offline Maybe include a short summary?